test/integration/GCHP/integrationTest.sh

```diff
@@ -178,6 +193,19 @@ fi
 logsDir="${itRoot}/${LOGS_DIR}"
 scriptsDir="${itRoot}/${SCRIPTS_DIR}"
 
+# If --no-bootstrap is selected, then add a line to integrateTestExecute.sh
+# that will tell GCHP to require all species in the restart file.
+if [[ "x${bootStrap}" == "xno" ]]; then
+    line="\# Edit setCommonRunSettings.sh to disable bootstrapping"
+    sed_ie "s/REPLACE1/${line}/" "${scriptsDir}/integrationTestExecute.sh"
+
+    line="sed_ie 's\/Require_Species_in_Restart\=0\/Require_Species_in_Restart\=1\/' setCommonRunSettings.sh"
+    sed_ie "s/REPLACE2/${line}/" "${scriptsDir}/integrationTestExecute.sh"
+else
+    sed_ie '/REPLACE1$/d' "${scriptsDir}/integrationTestExecute.sh"
+    sed_ie '/REPLACE2$/d' "${scriptsDir}/integrationTestExecute.sh"
+fi
```

Comment:
I'm not quite following how this works. Is there a way to have an if else based on the setting passed rather than use sed to insert lines?

Reply:
Hi @lizziel: The way I have it is that the updating of the data files is done in each run dir, which has to get done in the SLURM batch job via integrationTestExecute.sh.  Maybe I can pull that out of the execution scripts and into the run directory creation process.   Let me give that a think and I'll push an update later.

Reply:
Now fixed in commit fb09808.